cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -349,7 +356,44 @@ func (uc *UpstreamController) updateNodeStatus() {
 			}
 
 			switch msg.GetOperation() {
+			case model.InsertOperation:
+				_, err := uc.kubeClient.CoreV1().Nodes().Get(name, metaV1.GetOptions{})
+				if err == nil {
+					klog.Infof("node: %s already existed, do nothing", name)
```

Comment:
```suggestion
					klog.Infof("node: %s already exists, do nothing", name)
```

Reply:
@kevin-wangzefeng , I didn't deal with this suggested change in time before a force-push, so    I had to change it manually.